src/test/resources/logback.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Please, turn logging completely off in the test. You can always run the test with a different logback config when you are developing. I would like the build output to remain clean by default. For instance, if a test provokes an error on purpose, I don't want to see any error logging, as the "error" is actually "good" in that context.